client/src/GlobalStyle.js

```diff
@@ -8,13 +8,14 @@ export default createGlobalStyle`
 
     body {
         font-family: sans-serif;
-        display: grid;
-        height: 100%;
-
+        height: 100vh;
     }
 
     #root {
     display: grid; 
-    background: #e3e0de;  
+    grid-template-rows: 9vh auto;
+    overflow: scroll;
```

Comment:
Overflow: scroll ist sinnvoller in dem Element, in dem du etwas scrollbar machen möchtest, zB. dem main-Element